draft-bbs-signatures.md

```diff
@@ -761,6 +761,9 @@ A cryptographic hash function that takes as an arbitrary octet string input and
 
 - message_generator_seed: The seed used to generate the message generators which form part of the public parameters used by the BBS signature scheme, Note there are multiple possible scopes for this seed including; a globally shared seed (where the resulting message generators are common across all BBS signatures); a signer specific seed (where the message generators are specific to a signer); signature specific seed (where the message generators are specific per signature). The ciphersuite MUST define this seed OR how to compute it as a pre-cursor operations to any others.
 
+generator_no_of_bytes
+: Number of bytes to draw from the XOF when creating generators as per the operation documented in (#creategenerators). It is RECOMMENDED this value be set to one greater than `ceil(r+k)/8` for the ciphersuite, where `r` and `k` are parameters from the underlying pairing friendly curve being used.
```

Comment:
```suggestion
: Number of bytes to draw from the XOF when creating generators as per the operation documented in (#creategenerators). It is RECOMMENDED this value be set to one greater than `ceil(log2(p)+k)/8` for the ciphersuite, where `p` and `k` are parameters from the underlying pairing friendly curve being used.
```

No need to introduce a new parameter (r) IMO.

Reply:
```suggestion
: Number of bytes to draw from the XOF when creating generators as per the operation documented in (#creategenerators). It is RECOMMENDED this value be set to one greater than `ceil(r+k)/8` for the ciphersuite, where `p` the prime characteristic of the G1 and G2 subgroups and `k` the targeted security level (in bits) of the ciphersuite. 
```
Since we don't describe `p` and `k` anywhere else (they are defined in the ciphersuites we reference, however this must be fixed to be clearer, opened issue #118 for that) added here a suggestion for quick explanation of what they are. This may be nice to have even if #118 is closed, since those parameters will most likely only be defined as references.